portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -22,44 +22,45 @@
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. TODO: modify this file 
+to handle user input instead of hard-coded comments*/
 @WebServlet("/data")
 public final class DataServlet extends HttpServlet {
 
-	@Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        
-        PortfolioComments commentsRecord = new PortfolioComments();
+  /** Takes in hard-coded messages and store them in json **/
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  
+    PortfolioComments commentsRecord = new PortfolioComments();
+    StringBuilder json = new StringBuilder();
```

Comment:
This field is not used, please remove if not used.

Reply:
The HttpServletRequest request is used to run the goGet method, i tried deleting it and errors popped up when I ran it